src/Native/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ endfunction()
 
 if (CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL amd64)
     set(IS_64BIT_BUILD 1)
+    add_compile_options(-mcx16)
```

Comment:
It needs to be elsewhere, since it is Unix only option. Ideally in the block starting at line 63 with other Unix compiler options.